test/bin/manage_hypervisor_config.sh

```diff
@@ -8,6 +8,9 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
 
+VGFILE=/var/lib/microshift-lvm-storage.img
```

Comment:
I'm not sure this is the "right" place for the storage file. 
Could it be that it fits better in per-scenario directories? Otherwise, different tests can affect each other by adjusting the same file, right?

Reply:
Yeah, but the VGs are shared between host and containers.